adivinaLaPalabra/src/main/java/com/soprasteria/adivinaLaPalabra/config/WordsReader.java

```diff
@@ -27,10 +28,18 @@ private void convertJsonToList() {
     }
 
     public boolean checkWord(String word) {
-        return words.contains(word);
+        return dictionary.contains(word);
     }
 
-    public List<String> getWords() {
-        return words;
+    public List<String> getDictionary() {
+        return dictionary;
+    }
+
+    public String getRandomWord() {
+        Random random = new Random();
+        int dictionarySize = dictionary.size();
+        int randomIndex = random.nextInt(dictionarySize);
```

Comment:
```suggestion
        int randomIndex = random.nextInt(dictionary.size());
```
There is no need to define a new variable for size, call the size() method directly and remove the dictionarySize variable